api/types/backend/backend.go

```diff
@@ -0,0 +1,62 @@
+package backend
+
+import (
+	"io"
+	"net/http"
+	"time"
+)
+
+// ContainersConfig is the filtering specified by the user to iterate over containers.
+type ContainersConfig struct {
```

Comment:
Looks similar to https://github.com/docker/engine-api/blob/master/types/client.go#L44